lua/telescope/actions/set.lua

```diff
@@ -68,6 +68,7 @@ local edit_buffer
 do
   local map = {
     drop = "drop",
+    ["tab drop"] = "tab drop",
```

Comment:
We should use the same key everywhere: https://github.com/nvim-telescope/telescope.nvim/pull/2143/files#diff-536f772c8bf9390acea7443c9dc1d570cde4901c85d77ba1334d47e3239e242bR37 so `tab_drop` also i dont think you need to `["` `"]` the key anymore

Reply:
this doesn't seem to work when i change this to `tab_drop` unfortunately, though i am unsure why. i was under the impression these keys should be valid nvim commands though that just map to the local buffer cloning commands. for example, all of the existing keys are all runnable commands that just map to some local buffer cloning commands. perhaps there's some detail there that makes this not work as `tab_drop` (which is not a valid nvim command) as a result

i did update [this line](https://github.com/nvim-telescope/telescope.nvim/pull/2143/files#diff-536f772c8bf9390acea7443c9dc1d570cde4901c85d77ba1334d47e3239e242bR37) though to remove the surrounding `["`/`"]` though for that `tab_drop` command